### DISCUSSION TITLE
raspiberry 5 nvme compatible

### DISCUSSION THREAD

**Discussion by up1son** [raspiblitz/raspiblitz#4610] [Q&A]
Can anyone answer a question about running an nvme on Raspblitz? Is this compatible? SSD 2TB Crucial P3, M.2 2280, NVMe PCIe Gen 3 x4, Leitura 3500MB/s, Grav. 3000MB/s - CT2000P3SSD8 and  X1001 PCIe to M.2

**Comment by rootzoll** [comment]
SSD 2TB Crucial P3 - is running fine so far in tests
X1001 PCIe nvme board worked on test - but no longer endurenace testing so far
